welcome/run-d-program-locally.md

```diff
@@ -0,0 +1,40 @@
+# Eseguire programmi D sul proprio computer
+
+Il linguaggio D è accompagnato non solo dal compilatore `dmd`, ma anche dallo pseudo-interprete `rdmd` e dal gestore di pacchetti `dub`.
+
+### Il compilatore DMD
+
+Il compilatore *DMD* compila file D per produrre un file binario eseguibile.
+Sulla riga di comando *DMD* dev'essere invocato seguito dal/dai file da compilare:
+
+    dmd esempio.d
+
+Il comportamento del compilatore può essere modificato tramite le molte opzioni specificabili.
+L'elenco delle opzioni disponibili si può trovare nella [documentazione online](https://dlang.org/dmd.html#switches) o eseguendo `dmd --help`.
```

Comment:
It sounds pedantic. Maybe "[...] può essere modificato in molti modi. L'elenco delle opzioni... [...]" or "[...] le molte opzioni disponibili. L'elenco completo si può trovare [...]"